File: FizzBuzz.c
#include<stdio.h>
void main()
{
    int i,n;
    printf("Enter the limit ");
    scanf("%d",&n);
    for(i=1;i<=n;i++)
    {
        if(i%3!=0 && i%5!=0) 
        printf("%d\n", i);
        if(i%3 == 0 && i%5!=0)
        printf("Fizz\n");
        if(i%5 == 0 && i%3!=0)
        printf("Buzz\n");
        if(i%5 == 0 && i%3==0)
        printf("FizzBuzz\n");
    }
}

File: FizzBuzz.py
for num in range(1,101):
    if num % 15 == 0:  
        print(" FizzBuzz ")                                          
        continue
    elif num % 3 == 0:      
        print(" Fizz")                                          
        continue
    elif num % 5 == 0:          
        print(" Buzz ")                                      
        continue
    print(num)  
